runtime/lib/backend/lightning-kokkos/LightningKokkosSimulator.cpp

```diff
@@ -186,38 +186,6 @@ auto LightningKokkosSimulator::HamiltonianObservable(const std::vector<double> &
     return this->obs_manager.createHamiltonianObs(coeffs, obs);
 }
 
-// TODO: remove this kernel after merging expval(const ObservableKokkos<T> &ob)
```

Comment:
These kernels were added upstream to LKokkos and it's now part of the latest-release. Check [PR 47](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/47).